modules/apps/object/object-service/src/main/java/com/liferay/object/internal/servlet/filter/ObjectActionContextFilter.java

```diff
@@ -5,7 +5,7 @@
 
 package com.liferay.object.internal.servlet.filter;
 
-import com.liferay.object.action.util.ObjectActionThreadLocal;
+import com.liferay.object.action.util.HttpServletRequestThreadLocal;
```

Comment:
@victorKammerer also rename this class to `ObjectContextFilter` remember to change the `servlet-filter-name` as well